docs/contributor/arch/013-resource-attributes-enrichment-for-non-workload-metrics.md

```diff
@@ -53,17 +53,19 @@ Load test using the image `europe-docker.pkg.dev/kyma-project/dev/kyma-otel-coll
 | 1.Current Setup (Without Connectors) |             4458             |             4458             |          0          |       143, 162       |   1.5, 1.5    |             
 | 2.Current Setup (Without Connectors) |             4476             |             4476             |          0          |       138, 153       |   1.6, 1.5    |
 | 3.Current Setup (Without Connectors) |             4395             |             4396             |          0          |       146, 163       |   1.6, 1.5    |
-|                   1. With Connectors |             4149             |             4149             |          0          |       192, 163       |   1.7, 1.6    |
-|                   2. With Connectors |             4155             |             4156             |          0          |       174, 182       |   1.7, 1.6    |
-|                   3. With Connectors |             4069             |             4069             |          0          |       191, 181       |   1.6, 1.6    |
+|                   1. With Connectors |             4459             |             4461             |          0          |       172, 153       |   1.5, 1.5    |
+|                   2. With Connectors |             4476             |             4479             |          0          |       167, 138       |   1.5, 1.5    |
+|                   3. With Connectors |             4471             |             4471             |          0          |       157, 163       |   1.5, 1.5    |
+
 
 </div>
 
 - _Pros_: 
   - Clean solution. The non-workload metrics will never have the unwanted resource attributes set to any value.
   - This is the recommended solution for doing a conditional routing in an OTel collector pipeline.
+  - If a user sends their own custom non-workload metrics, they can skip enriching their custom non-workload metrics with unwanted resource attributes by setting a resource attribute (like `skip-enrichment`) to `true`. This will be documented for the users, so that they would be aware of the possibility of skipping the resource attributes enrichment.
 - _Cons_:
-  - Per MetricPipeline, we will have 3 pipelines in the collector instead of 1. So, we will have a more complex setup of the pipeline definitions in combination with the new connectors definitions.
+  - Per MetricPipeline, we will have 3 pipelines in the collector instead of 1. So, we will have a more complex setup of the pipeline service definitions in combination with the new connectors definitions.
```

Comment:
```suggestion
  - Per MetricPipeline, we will have three pipelines in the collector instead of one. So, we will have a more complex setup of the pipeline service definitions in combination with the new connectors definitions.
```